Lib/ufo2ft/featureWriters/markFeatureWriter.py

```diff
@@ -34,7 +34,7 @@ def warnIfAmbiguous(self, log):
                 markGlyphToMarkClasses[markGlyph].add(namedAnchor.markClass.name)
         for markGlyph, markClasses in markGlyphToMarkClasses.items():
             if len(markClasses) > 1:
-                log.warning(
+                log.info(
                     "The base glyph %s and mark glyph %s are ambiguously "
                     "connected by several anchor classes: %s. "
                     "Only one will prevail; which exactly is not guaranteed.",
```

Comment:
> Only one will prevail; which exactly is not guaranteed.

Well, we do know which one it will prevail: the last one. I think it's better in the logging message to print out the order in which we will encode the lookups and remind that the last one is the one that will prevail.

Reply:
Done